src/App.tsx

```diff
@@ -1,26 +1,203 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import * as todoMethods from './api/todo';
+import { ErrorNotification } from './components/Error';
+import { FilterStatus } from './types/FilterStatus';
+import getTodosFilter from './utils/getTodosFilter';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterStatus, setFilterStatus] = useState<FilterStatus>(
+    FilterStatus.All,
+  );
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [deletingTodoIds, setDeletingTodoIds] = useState<number[] | null>(null);
+  const [updatingTodoIds, setUpdatingTodoIds] = useState<number[] | null>(null);
```

Comment:
what is the different between this states?
can we use just one?
like processingIds or smth like that